src/openzaak/conf/includes/base.py

```diff
@@ -502,18 +504,24 @@
 #
 SENTRY_DSN = config("SENTRY_DSN", None)
 
+SENTRY_SDK_INTEGRATIONS = [
+    django.DjangoIntegration(),
+    redis.RedisIntegration(),
+]
+
 if SENTRY_DSN:
-    INSTALLED_APPS = INSTALLED_APPS + ["raven.contrib.django.raven_compat"]
-
-    RAVEN_CONFIG = {"dsn": SENTRY_DSN, "release": GIT_SHA}
-    LOGGING["handlers"].update(
-        {
-            "sentry": {
-                "level": "WARNING",
-                "class": "raven.handlers.logging.SentryHandler",
-                "dsn": RAVEN_CONFIG["dsn"],
-            }
-        }
+    import sentry_sdk
+
+    SENTRY_CONFIG = {
+        "dsn": SENTRY_DSN,
+        "release": os.getenv("RELEASE", "RELEASE not set"),
```

Comment:
The RELEASE variable is already initialized. Why grab it from the env again and miss out on the GIT_SHA alternative?

```suggestion
        "release": RELEASE or "RELEASE not set",
```